assets/src/components/security/policies/CreatePolicyReportModal.tsx

```diff
@@ -22,7 +22,15 @@ const fetchPolicyReport = (format: ReportFormat, token: string) => {
     method: 'GET',
     headers: { Authorization: `Bearer ${token}` },
   }).then((res) => {
-    const writeStream = streamSaver.createWriteStream('report.zip')
+    const filename =
```

Comment:
is there maybe a npm package or pre-made impl for this? At least to parse the headers first then maybe you can parse the content disposition from there

Reply:
There is https://github.com/jshttp/content-disposition. I will use it.